documentation/endpoints/kudos.md

```diff
@@ -20,16 +21,75 @@ GET https://www.dosomething.org/api/v1/kudos
 
 ### Optional query parameters
 
+**NOTE: The following query parameters no longer work with this endpoint and will soon be deprecated or refactored!**
```

Comment:
"no longer work with this endpoint"? Should they be removed from the docs, or am I misunderstanding?